commands.go

```diff
@@ -29,10 +29,6 @@ type AppendEntriesResponse struct {
 
 	// We may not succeed if we have a conflicting entry
 	Success bool
-
-	// There are scenarios where this request didn't succeed
-	// but there's no need to wait/back-off the next attempt.
-	NoRetryBackoff bool
```

Comment:
Removed this because I don't think there are legitimate scenarios where the leader *should* back off upon receiving a reply. Backoff happens now only in response to transport errors.